src/ucp/core/ucp_worker.c

```diff
@@ -1219,7 +1220,9 @@ void ucp_worker_iface_cleanup(ucp_worker_iface_t *wiface)
         }
     }
 
-    ucp_worker_uct_iface_close(wiface);
+    if (wiface->iface != NULL) {
+        ucp_worker_uct_iface_close(wiface);
```

Comment:
seems like for same reason, ucp_worker_uct_iface_close() should check it: if a function receives an object, it should check its fields internally.

Reply:
done